packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-actions.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  TestFixture,
+  addTag,
+  TagProps,
+  generateComponentHTML,
+} from '../../../fixtures/test-fixture';
+import {NumericFacetSelectors} from './numeric-facet-selectors';
+
+export interface NumericRange {
+  start: number;
+  end: number;
+}
+export const defaultNumberOfValues = 8;
+export const label = 'Youtube Views';
+export const field = 'ytviewcount';
+export const numericRanges: NumericRange[] = [
```

Comment:
Can we leverage `endInclusive` instead of bumping the start value of the next range?